voter_info/districts/models.py

```diff
@@ -6,28 +6,40 @@
 from django.contrib.gis.geos import GEOSGeometry
 
 
+# a custom queryset and manager to be able to write:
+# District.objects.contains_point(lat, lon) -> QuerySet[ <Area 1>, <Area 2>,... ]
+class DistrictQuerySet(models.QuerySet):
+    def contains_point(self, lat, lon):
+        return self.filter(areas__mpoly__contains=GEOSGeometry(f'POINT({lon} {lat})')).distinct('id')
+
+
+class DistrictManager(models.Manager):
+    def get_queryset(self):
+        return DistrictQuerySet(self.model, using=self._db)
+
+    def contains_point(self, lat, lon):
+        return self.get_queryset().contains_point(lat, lon)
+
+
```

Comment:
moving the `contains_point` method from a `@classmethod` on `District` into the `DistrictQuerySet` lets you chain the `contains_point` method like you would any other filter.

e.g. instead of:

```python
District.contains_point(lat, lon)
```

You can apply `contains_point` to existing querysets, e.g.

```python
queryset = District.objects.filter(election_date__gte=some_date).contains_point(lat, lon)
```